### PR TITLE
Add docs for setting up mysql 5.7 for Mac OS X.

### PR DESCRIPTION
**Pull request by Seeker1911**

### CODE REVIEW

File: rest-api/README.md [hunk 1]
```diff
@@ -20,11 +20,17 @@ Make sure that you have google
 [cloud SDK](https://cloud.google.com/sdk/downloads) installed.
 
 From the rest-api directory, run:
-
+#### For Ubuntu
 * `sudo apt-get install mysql-server libmysqlclient-dev` (to install MySQL server and client —
 if you prefer to use docker, see note below)
+#### For Mac OS
+* `brew isntall mysql-connector-c`
```
Comment (danqrodney) [inline rest-api/README.md]: istnall -> install

File: rest-api/README.md [hunk 2]
```diff
@@ -20,11 +20,17 @@ Make sure that you have google
...
+* If mysql is installed via homebrew you may have to symlink the mysql client:
```
Comment (danqrodney) [inline rest-api/README.md]: Do you want to describe how to install mysql via homebrew? Is the above command all you have to do?

File: rest-api/README.md (hunk 1 above)
Comment (Seeker1911) [reply rest-api/README.md]: done

File: rest-api/README.md (hunk 2 above)
Comment (Seeker1911) [reply rest-api/README.md]: done

**Pull request merged**